templates/index.html.twig

```diff
@@ -3,5 +3,30 @@
 {% block title %}Homepagina{% endblock %}
```

Comment:
Since we now have a working homepage, could you remove this file and, if needed, move this to the home page ?

Reply:
you could if you want to yes.